PyISY/constants.py

```diff
@@ -399,3 +415,38 @@
     },
     "99": {"7": "on", "8": "auto"},  # Insteon Thermostat Fan Mode
 }
+
+INSTEON_RAMP_RATES = {
```

Comment:
Is this just here for consumers to import and use to understand how to interpret ramp rates in aux_properties?

It seems to me that we should change this to be a number representing the how many seconds the ramp rate is. That'll make working with the values easier (and more localizable)

Reply:
That was moved from the Home Assistant integration, I had left it there for a user to be able to translate where possible, but now I agree that it should actually be useful if we're going to include it.

I updated the aux_properties functions to translate the index to a true value in seconds.

Reply:
Thinking about more, should that be added as a separate property to not break anyone's integration (looking for and translating "RR" themselves? Or should we just document it as a breaking change?

Reply:
I think we should document it as a breaking change, but open to disagreement :)

Reply:
Well, I guess, considering RR is not reported in the initial query, and it was never stored in aux properties before, and I was the one that implemented it on the HA end to store a translated version... I'd say we should be safe.

Thinking about it, it's probably not even a breaking change... The RR event can still be picked up separately from the event stream.